modules/apps/click-to-chat/click-to-chat-web/src/main/resources/META-INF/resources/dynamic_include/zendesk.jsp

```diff
@@ -16,14 +16,44 @@
 
 <%@ include file="/dynamic_include/init.jsp" %>
 
-<script id="ze-snippet" src="https://static.zdassets.com/ekr/snippet.js?key=<%= clickToChatChatProviderAccountId %>">
-</script>
-
-<c:if test="<%= themeDisplay.isSignedIn() %>">
-	<script>
-		zE('webWidget', 'identify', {
-			email: '<%= user.getEmailAddress() %>',
-			name: '<%= user.getScreenName() %>',
-		});
-	</script>
-</c:if>
\ No newline at end of file
+<script>
+	function loadScript() {
```

Comment:
I see this function `loadScript` inserted in the Global DOM with a very generic name could be bad, in other files we have this same loadScript name.

This could cause a function conflict or further bad functional.

In preview review last time he suggested us to use <portlet:namespace /> to avoid this to happen.

something like:

`function loadScript() {`
`function <portlet:namespace />loadScript() {` 

I know is not a fancy name, but help a lot to avoid these name collisions

Other thing could be better is, loadScript is also very generic, maybe something like:

`function <portlet:namespace />loadZendeskScript() {`

You can call the function this way:

`<portlet:namespace />loadZendeskScript();`

```suggestion
	function <portlet:namespace />loadZendeskScript() {
```